app/controllers/stories_controller.rb

```diff
@@ -143,7 +164,7 @@ def validate_url_product_id
   end
 
   def stories_params
-    params.require(:story).permit(:title, :description, :extra_info, :project_id)
+    params.require(:story).permit(:title, :description, :extra_info, :project_id, :approved)
```

Comment:
shouldn't this be status?

```suggestion
    params.require(:story).permit(:title, :description, :extra_info, :project_id, :status)
```